### PR TITLE
Change UI that Change Operator.

### PR DESCRIPTION
**Pull request by satellitex**
Depends on this PR.
(https://github.com/staketechnologies/Plasm/pull/87)

Now, not applied `BTreeSet` on PolkadotJS. So temporary uses `Vec`.

### CODE REVIEW

File: packages/apps/webpack.config.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
       new HtmlWebpackPlugin({
         inject: true,
         template: path.join(context, `${hasPublic ? 'public/' : ''}${name}.html`),
-        PAGE_TITLE: 'Polkadot/Substrate Portal'
+        PAGE_TITLE: 'Plasmchain Portal'
```
Comment (akru) [inline packages/apps/webpack.config.js]: What’s about just Plasm or Plasm Network?

File: packages/app-accounts/src/modals/ChangeOperator.tsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    } else {
+      setHasAvailable(false)
+    }
+  }, [selects, recipientId, senderId]);
+
+  const transferrable = <span className='label'>{t('transferrable')}</span>;
+
+  return (
+    <Modal
+      className='app--accounts-Modal'
```
Comment (akru) [inline packages/app-accounts/src/modals/ChangeOperator.tsx:119]: Is this class name uniq?

File: packages/apps/webpack.config.js (hunk 1 above)
Comment (satellitex) [reply packages/apps/webpack.config.js]: Plasm Network is fixed.

File: packages/app-accounts/src/modals/ChangeOperator.tsx (hunk 2 above)
Comment (satellitex) [reply packages/app-accounts/src/modals/ChangeOperator.tsx:119]: Other modal uses the class name.

**Review by akru — APPROVED** [review]

**Pull request merged**